plugins/generator-1.20.1/forge-1.20.1/mappings/blockstateproperties.yaml

```diff
@@ -0,0 +1,305 @@
+ATTACHED:

```

Comment:
I believe you omitted waterlogged, for example, because MCreator defines it. But maybe there is a chance one wants to do custom waterlogged or axis or facing implementation, but right now this PR does not enable that.

I would add those properties here too and make sure validation handles them correctly if waterlogging is already enabled by MCreator, for example.. same with facing, axis, ...

Reply:
Custom implementation of e.g. facing enum will not work with vanilla rotation functions because it is a different enum class, for example

Reply:
`facing` and `axis` are special subclass of `EnumProperty<Direction>` so not sure if I can/should include these ones back.

Reply:
I don't think anything should break, they are still EnumProperty by default.

Limiting users to not be able to use them for custom rotation logic may not be the best choice.

Reply:
:heavy_check_mark: Fixed